src/main/kotlin/uk/gov/justice/digital/assessments/jpa/entities/AssessmentEntity.kt

```diff
@@ -28,7 +28,7 @@ class AssessmentEntity(
   val assessmentUuid: UUID = UUID.randomUUID(),
 
   @Column(name = "CREATED_DATE")
-  val createdDate: LocalDateTime? = null,
+  val createdDate: LocalDateTime = LocalDateTime.now(),
 
```

Comment:
createDate is not nullable

CREATE TABLE IF NOT EXISTS assessment
(
    assessment_id           SERIAL          PRIMARY KEY,
    assessment_uuid         UUID            UNIQUE,
    created_date            TIMESTAMP       NOT NULL,
    completed_date          TIMESTAMP
);